Description: Relax dependecy on tagged
 In is not clear why, but cabal-unpack download binary-orphans
 version 0.1.2 with `x-revision: 1', where dependency on tagged
 is relaxed. Unfortunatelly, uscan download tarball without
 x-revision, so this patch is needed.
 .
 In next, 0.1.3 upstream version problem is solved, so patch do
 not need to be forwarded upstream.
Author: Dmitry Bogatov <KAction@gnu.org>
Forwarded: no
Last-Update: 2015-12-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/binary-orphans.cabal
+++ b/binary-orphans.cabal
@@ -35,7 +35,7 @@ library
     , binary                   >=0.7  && <0.7.7
     , hashable                 >=1.2  && <1.3
     , scientific               >=0.3  && <0.4
-    , tagged                   >=0.7  && <0.8.2
+    , tagged                   >=0.7  && <0.8.3
     , text                     >=1.2  && <1.3
     , time                     >=1.4  && <1.5.1
     , unordered-containers     >=0.2  && <0.3
@@ -58,7 +58,7 @@ test-suite binary-orphans-test
     , binary                   >=0.7  && <0.7.7
     , hashable                 >=1.2  && <1.3
     , scientific               >=0.3  && <0.4
-    , tagged                   >=0.7  && <0.8.2
+    , tagged                   >=0.7  && <0.8.3
     , text                     >=1.2  && <1.3
     , time                     >=1.4  && <1.5.1
     , unordered-containers     >=0.2  && <0.3
